streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -736,6 +735,26 @@ private InternalConsumerConfig(final Map<String, Object> props) {
         }
     }
 
+    private static String getTaskProducerClientId(final String threadClientId, final TaskId taskId) {
+        return threadClientId + "-" + taskId + "-producer";
+    }
+
+    private static String getThreadProducerClientId(final String threadClientId) {
+        return threadClientId + "-producer";
+    }
+
+    private static String getThreadConsumerClientId(final String threadClientId) {
```

Comment:
nit: I guess this and the next one could be `getConsumerClientId` and `getRestoreConsumerId` as there is only ever one of each?

Reply:
ack.